### PR TITLE
Touch images.txt in build process if it doesn't exist

### PR DESCRIPTION
**Pull request by avatarneil**
## 📚 Purpose
Fixes build issue

### CODE REVIEW

File: bin/build-helper [hunk 1]
```diff
@@ -21,6 +25,10 @@ buildImage() {
 }
 
 pushImages() {
+  if [! -f $BASE_DIR/dist/images.txt]
```
Comment (drtyh2o) [inline bin/build-helper:28]: I am curious about this. It looks like line `32` is going to empty the `images.txt` file. Line `36` seems like it expects to read the values written by `buildImage` but I'm not totally tracking.

Comment (avatarneil) [reply bin/build-helper:28]: Yeah, I'm not too clear on the expected functionality here either (eternal bash noob), I just wanted to verify that any time we're interacting with the file it's "safe"

**Review by drtyh2o — APPROVED** [review]

File: bin/build-helper [hunk 2]
```diff
@@ -8,8 +8,19 @@ BASE_DIR=$(dirname $0)/..
 NODE_ENV=${NODE_ENV:-""}
 REGISTRY=${REGISTRY:=""}
 
+createImagesFileIfNotExists() {
+  if [ ! -d $BASE_DIR/dist ]; then
```
Comment (invertigo) [inline bin/build-helper:12]: https://google.github.io/styleguide/shellguide.html#s6.3-tests

TL; DR - use `[[`.

**Review by invertigo — CHANGES_REQUESTED** [review]

**Review by invertigo — APPROVED** [review]
